src/js/components/molecules/list-section/list-section.vue.html

```diff
@@ -29,6 +29,10 @@ <h3 class="list-section__title">{{ title }}
         </header>
     </template>
 
+    <template v-if="filterCategories" v-for="filterCategory in filterCategories">
```

Comment:
`v-for` and `v-if` should not be used on the same element… https://vuejs.org/v2/guide/conditional.html#v-if-with-v-for

Reply:
ja, not sure how this one slipped in. thanks for catching it.